src/main/kotlin/io/emeraldpay/dshackle/config/UpstreamsConfig.kt

```diff
@@ -119,7 +120,24 @@ open class UpstreamsConfig {
 
     class EthereumConnection : RpcConnection() {
         var ws: WsEndpoint? = null
-        var preferHttp: Boolean = false
+        var preferHttp: Boolean? = null
```

Comment:
Why `preferHttp` is become nullable?

Reply:
we need to determine whether there was a value or not

Reply:
I really don't see the usage of null `preferHttp`, it seems to me that non-null covers all cases